### PR TITLE
[feature1] first feature was implemented

### PR DESCRIPTION
**Pull request by warriorIndeed**
Following changes were made: 

new-feature.txt was added
feature1 folder was created

### CODE REVIEW

File: feature1/second.txt
Comment (warriorIndeed) [inline feature1/second.txt:1]: It's a comment ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Comment (vypaliuvach) [inline feature1/second.txt:1]: It's a comment by vypaliuvach

Comment (vypaliuvach) [reply feature1/second.txt:1]: It's a comment by vypaliuvach~~~

File: new-feature.txt
Comment (vypaliuvach) [inline new-feature.txt:1]: It's a comment for new-feature.txt by vypaliuvach

Comment (warriorIndeed) [reply new-feature.txt:1]: Thanks for comments, vypaliuvach!

**Review by vypaliuvach — APPROVED** [review]
Good job, warriorIndeed

**Pull request merged**